src/app/pages/website/blog/blog.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'redirect',
+  template: 'redirecting...'
+})
+export class BlogComponent implements OnInit {
+  constructor() { }
+
+  ngOnInit() {
+    window.location.href = 'https://www.devoxx4kids.org/nederland/'
```

Comment:
Nit: semi-colon missing at the end-of-line

Reply:
semi-colon added